courses/templates/course_detail.html

```diff
@@ -95,4 +94,27 @@ <h3>{{ course.outcomes.sub_heading | safe }}</h3>
      </div>
  {% endif %}
 </div>
+  {% if course.who_should_enroll %}
+    <div class="who-should-enroll-block">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-6">
+            <h1>{{ course.who_should_enroll.title }}</h1>
```

Comment:
I think, we can keep it hard coded. `WHO SHOULD ENROLL` here.

Reply:
Done.